vectorstores/mongovector/doc.go

```diff
@@ -1,5 +1,7 @@
 // Package mongovector implements a vector store using MongoDB as the backend.
 //
+// Notice: this package uses the 2.0.0-beta1 release of the MongoDB Go Driver.
```

Comment:
```suggestion
// Notice: this package uses a beta release of the MongoDB Go Driver 2.0.0.
```
This way it's accurate whether #1026 gets merged or not.